corehq/motech/repeaters/const.py

```diff
@@ -6,6 +6,7 @@
 
 MAX_RETRY_WAIT = timedelta(days=7)
 MIN_RETRY_WAIT = timedelta(minutes=60)
+RATE_LIMITER_DELAY_RANGE = (timedelta(minutes=0), timedelta(minutes=15))
```

Comment:
What is the rationale for setting the max value in this range to 15 minutes? Only asking since prior to this change, repeat records would be postponed by 60 minutes.

Reply:
Yeah fair question. There's nothing scientific about how I chose these numbers. This just lets us guarantee that it'll be retried within 15 minutes, so there won't be major interruptions from minor overages. Not sure how necessary that is honestly, but I figured I'd try low like this and if the frequency of retrying itself (right now it'd average every 7.5m) becomes a problem we can up it.

Reply:
Yeah that sounds reasonable. Thanks